### DISCUSSION TITLE
Replacing own wildcard certificate

### DISCUSSION THREAD

**Discussion by SJD-DK** [globaleaks/globaleaks-whistleblowing-software#4109] [Q&A]
### What version of GlobaLeaks are you using?

Software version: 4.13.21

### What browser(s) are you seeing the problem on?

Microsoft Edge

### What operating system(s) are you seeing the problem on?

Windows

### Describe the issue

When I try to replace the wildcard certificate currently being used, I remove the old and try to upload the new wildcard certificate.
But the upload doesn't take place, when I refresh the page, the system no longer have a wildcard certificate.

Do you have a guide for replacing wildcard certificate?
What name extension should the certificate have?

**Comment by evilaliv3** [comment]
Thank you @SJD-DK 

Which is the format of the new certificate that you are trying to load? GlobaLeaks accepts only files in the PEM format.

I suggest you could try to update the system to the latest release since almost one year passed since the the release of the version that you are using.
